site.srht.shrimple.ouch.json

```diff
@@ -0,0 +1,90 @@
+{
+	"id": "site.srht.shrimple.ouch",
```

Comment:
`shrimple.shrt.site` doesn't seem to exist to provide a confirmation it is what it is.